frontend/src/components/Header.js

```diff
@@ -14,7 +14,7 @@ function Header({ username }) {
           <AppHeader username={username} />
         </Route>
         <Route path="/">
-          <img id="logo" src={logo} alt="BudgetCtrl logo" className="pb-3" />
+          <img id="logoWelcome" src={logo} alt="BudgetCtrl logo" className="pb-3" />
```

Comment:
Is the id change needed? Img size can be set here inline as well (using the height and width attributes). If you change ids, you need to change them also in the related MPV.js tests, they'll start failing otherwise.

Reply:
change made: worked infact w only height, width not needed